.pylintrc

```diff
@@ -0,0 +1,378 @@
+[MASTER]
```

Comment:
You used a default configuration file?

I think this is a good start. Let us continue with it.